### ISSUE TITLE
'help' and 'version' should be first class commands

### ISSUE THREAD

**Issue by markshuttle** [lxc/lxd#4381]
Please make 'lxc help' do the same thing as 'lxc --help'. Also, 'lxc version' should do the same thing as 'lxc --version'

**Comment by stgraber** [comment]
`help` should already be behaving the exact same as `--help` or as not passing any sub-command, that's the default behavior of the command line parser we use.

We'll re-add our own copy of the version subcommand as the parser only provides us with a `--version` argument and not a subcommand.

**Comment by stgraber** [comment]
```
stgraber@castiana:~/data/code/lxc/lxd (stgraber/master)$ lxc version
Client version: 3.0.0
Server version: 3.0.0.beta7
```

**Comment by stgraber** [comment]
Sent a pull request which addresses this issue. After it's reviewed and merged, the fix will be included in the stable-3.0 branch and be included in the candidate snap as well as in the next batch of cherry-picks for the LXD deb.
